README.md

```diff
@@ -40,3 +40,29 @@ the containing `x` package and their APIs will be locked.
 
 [cov-img]: https://codecov.io/gh/yarpc/yarpc-go/branch/master/graph/badge.svg
 [cov]: https://codecov.io/gh/yarpc/yarpc-go/branch/master
+
+## Development
+
+### Setup
+
+To start developing with yaprc-go, run the following command to setup your environment:
+
+```
+cd $GOPATH/src/github.com
+git clone https://github.com/yarpc/yarpc-go.git yarpc/yarpc-go
```

Comment:
This should actually be
```suggestion
cd $GOPATH/src/
git clone https://github.com/yarpc/yarpc-go.git go.uber.org/yarpc
```
since YARPC uses a canonical import path. Otherwise this could lead to compilation issues
- https://github.com/yarpc/yarpc-go/blob/dev/version.go#L21
- https://golang.org/doc/go1.4#canonicalimports